src/components/app/TabBar.js

```diff
@@ -51,6 +52,21 @@ class TabBar extends PureComponent {
   }
 }
 
+type Props = {
+  className?: string,
+  tabs: Tab[],
+  selectedTabName: string,
+  tabOrder: number[],
+  onSelectTab: string => mixed,
+  onChangeTabOrder: (number[]) => any,
+  +_mouseDownListener?: () => mixed,
+};
+
+type Tab = {
```

Comment:
No problem. I should have noticed, but I didn't really look around (besides the other PR). I went with my kneejerk reaction of keeping the duplicate definitions colocated.